src/index.js

```diff
@@ -1,4 +1,5 @@
 /** Brand */
+import SummaryList from "./components/site/SummaryList";
```

Comment:
this should be

```js
export { default as SummaryList } from "./components/site/SummaryList";
```

and should be moved to the site section.

Reply:
done